task/git-clone/0.1/git-clone.yaml

```diff
@@ -5,7 +5,7 @@ metadata:
   labels:
     app.kubernetes.io/version: "0.1"
   annotations:
-    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/pipelines.minVersion: "0.14.0"
```

Comment:
Should we move to 0.2 version of task then?

Reply:
Yes, probably this can be moved to 0.2 version. Will update the PR.

Reply:
not that there is https://github.com/tektoncd/catalog/pull/433 too (cc @sbwsg)